prow/plugins/lgtm/lgtm.go

```diff
@@ -202,3 +224,44 @@ func handlePullRequest(gc ghLabelClient, pe github.PullRequestEvent, log *logrus
 	}
 	return nil
 }
+
+func skipCollaborators(config *plugins.Configuration, org, repo string) bool {
+	full := fmt.Sprintf("%s/%s", org, repo)
+	for _, elem := range config.Owners.SkipCollaborators {
+		if elem == org || elem == full {
+			return true
+		}
+	}
+	return false
+}
+
+func loadRepoOwners(gc githubClient, ownersClient *repoowners.Client, org, repo string, number int) (*repoowners.RepoOwners, error) {
+	pr, err := gc.GetPullRequest(org, repo, number)
+	if err != nil {
+		return nil, err
+	}
+	return ownersClient.LoadRepoOwners(org, repo, pr.Base.Ref)
```

Comment:
Would we want to add support for `CODEOWNERS` files?

Reply:
Yeah, the owners client could be extended but orthogonal to the changes here.